src/ctia/http/routes/exploit_target.clj

```diff
@@ -69,16 +74,19 @@
                 :header-params [api_key :- (s/maybe s/Str)]
                 :summary "List ExploitTarget by external id"
                 :capabilities #{:read-exploit-target :external-id}
+                :middleware [wrap-un-store wrap-cache-control]
                 (paginated-ok
                  (page-with-long-id
                   (read-store :exploit-target list-exploit-targets
                               {:external_ids external_id} q))))
+
            (GET "/search" []
                 :return (s/maybe [StoredExploitTarget])
```

Comment:
I wonder how this PR doesn't break the tests.
Here in this route you state that it shall return a vector of `StoredExploitTarget`.
`StoredExploitTarget`requires the owner field.
so either the middleware somehow disables response schema validation or it never worked.